pkg/distributor/ha_tracker_test.go

```diff
@@ -120,19 +129,22 @@ func TestWatchPrefixAssignment(t *testing.T) {
 	codec := codec.Proto{Factory: ProtoReplicaDescFactory}
 	mock := kv.PrefixClient(consul.NewInMemoryClient(codec), "prefix")
 	c, err := newClusterTracker(HATrackerConfig{
-		EnableHATracker: true,
-		KVStore:         kv.Config{Mock: mock},
-		UpdateTimeout:   time.Millisecond,
-		FailoverTimeout: time.Millisecond * 2,
+		EnableHATracker:        true,
+		KVStore:                kv.Config{Mock: mock},
+		UpdateTimeout:          time.Millisecond,
+		UpdateTimeoutJitterMax: 0,
+		FailoverTimeout:        time.Millisecond * 2,
 	})
 	assert.NoError(t, err)
 
 	// Write the first time.
+	mtime.NowForce(start)
```

Comment:
was there something that necessitated this change?

Reply:
It avoids flaky tests.